Pat.Sender/Pat.Sender.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
The test project still uses net462, probs needs to be net6.0 and then I think your appveyor build might pass. For some reason I'm getting the emails 😂